packages/ui/src/components/DataStory/DataStory.tsx

```diff
@@ -77,16 +74,14 @@ export const DataStoryComponent = (
               <Allotment.Pane minSize={300} className="h-full w-96">
                 <DataStoryCanvas
                   {...props}
-                  onSave={() => new Promise(r => r)}
-                  key={diagramKey}
+                  onSave={client.updateTree}
+                  key={'data-story-canvas'}
```

Comment:
Will this static key be OK for docs site where many diagrams will be present on same page?

Reply:
In React, sibling elements at the same level cannot have duplicate key values. The key's primary function is to help React identify which elements are new, removed, or updated.

However, using the same key value across different component instances is fine. So, having identical keys within different DataStory components is not an issue.